### ISSUE TITLE
A11Y

### ISSUE THREAD

**Issue by shijinupc** [carbon-design-system/carbon-components-react#522]
## Detailed description

Describe in detail the issue you're having. Is this a feature request (new component, new icon), a bug, or a general issue?

> Is this issue related to a specific component?
Dropdown

> What did you expect to happen? What happened instead? What would you like to see changed?
second layer ul should have a menu role to contain li with menu item role

## Additional information
<img width="870" alt="00136aaa-ebee-11e7-890c-9f900bc4c1c9" src="https://user-images.githubusercontent.com/8380554/35210316-b31ebe7c-ff8c-11e7-981c-6725d295aef6.png">

**Comment by shijinupc** [comment]
Please also add an interface to accept Strings to generate an accessible label for the ul with menu role
